lxd/instance/drivers/driver_qemu.go

```diff
@@ -711,6 +711,7 @@ func (vm *qemu) Start(stateful bool) error {
 		"-nodefaults",
 		"-no-reboot",
 		"-no-user-config",
+		"-sandbox", "on,obsolete=deny,elevateprivileges=allow,spawn=deny,resourcecontrol=deny",
```

Comment:
@stgraber what was the elevateprivileges=allow needed for out of interest? Might be good to add a comment there for future reference.

Reply:
@stgraber was it just so that qemu can start as root and then drop its uid/gid to `lxd` user?

Reply:
Yeah, otherwise setuid/setgid is blocked